pygmt/tests/test_figure.py

```diff
@@ -121,6 +121,9 @@ def test_figure_show():
 def test_shift_origin():
     "Test if fig.shift_origin works"
     fig = Figure()
+    # First call shift_origin without -J & -R.
```

Comment:
```suggestion
    # First call shift_origin without projection and region.
```

Use long alias here? Maybe change the whole test to use projection and region instead of J and R, also frame instead of B.

Reply:
Good point. I also fixed the short arguments in other functions.